src/server/mod.rs

```diff
@@ -57,6 +57,7 @@ impl<T: Send + Debug + 'static> OnResponse<T> {
                 let _ = sender.send(resp);
             }
             OnResponse::Streaming(sender) => {
+                // `stream::once` never blocks so that we can safely `wait()` here.
```

Comment:
Can `send_all` block?

Reply:
our sender is a mpsc::channel's Sender so that it should not be a concern.

Reply:
Seems also need to be stated in comment.